index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+
+  <body>
+    <main id="main-doc">
+
+      <nav id="navbar">
+        <h1 id="nav-header">JS Documentation</h1>
+        <a href="#Introduction" class="nav-link">Introduction</a>
+        <a href="#What_you_should_already_know"  class="nav-link">What you should already know</a>
+        <a href="#JavaScript_and_Java"  class="nav-link">JavaScript and Java</a>
+        <a href="#Hello_World" class="nav-link" >Hello World</a>
+        <a href="#Variables" class="nav-link">Variables</a>
+        <a href="#Declaring_variables" class="nav-link">Declaring variables</a>
+        <a href="#Variable_scope" class="nav-link">Variable scope</a>
+        <a href="#Global_variables" class="nav-link">Global variables</a>
+        <a href="#Constants" class="nav-link">Constants</a>
+        <a href="#Data_types" class="nav-link">Data types</a>
+        <a href="#If...else_statement" class="nav-link">If...else statement</a>
+        <a href="#While_statement" class="nav-link">While statement</a>
+        <a href="#Function_declarations" class="nav-link">function declarations</a>
+        <a href="#Reference" class="nav-link">Reference</a>
+      </nav>
+    
+
+      <div id="sections">
```

Comment:
Please use semantic elements for declaring sections rather than divs for better clarity and readability